tests/core/tests/test_model_resource_fields_generate_widgets.py

```diff
@@ -0,0 +1,199 @@
+from unittest import TestCase
+
+import django
+from django.contrib.contenttypes import fields as contenttype_fields
+from django.contrib.postgres import fields as postgres
+from django.contrib.postgres import search as postgres_search
+from django.contrib.postgres.fields import ranges as postgres_ranges
+from django.db import models
+from django.db.models.fields.related import RelatedField
+
+from import_export import widgets
+from import_export.resources import ModelResource
+
+from ..models import WithPositiveIntegerFields
+
+
+class ExampleResource(ModelResource):
+    class Meta:
+        model = WithPositiveIntegerFields
+
+
+class TestImportExportBug(TestCase):
```

Comment:
Please could you rename this to a more descriptive term?  Perhaps `TestFieldWidgetMapping`.